node/cmd/guardiand/node.go

```diff
@@ -373,6 +384,16 @@ func runNode(cmd *cobra.Command, args []string) {
 		logger.Fatal("Please specify --terraContract")
 	}
 
+        if *algorandRPC == "" {
+                logger.Fatal("Please specify --algorandRPC") 
+        }
+        if *algorandToken == "" { 
+                logger.Fatal("Please specify --algorandToken") 
+        }
+        if *algorandContract == "" { 
+                logger.Fatal("Please specify --algorandContract") 
+        }
+
```

Comment:
Best to put this behind a `*unsafeDevMode` condition such that it only runs in devnet.

(also, the formatting is wrong - Go uses exclusively tabs - just have your editor run `gofmt` on save and it fixes it up automatically)

Reply:
Can you send me your emacs settings you use for go?

Reply:
Well... I don't anymore, just use Emacs for org-mode these days and IntelliJ for everything else :P